fastly/client.go

```diff
@@ -275,6 +290,16 @@ func (c *Client) Request(verb, p string, ro *RequestOptions) (*http.Response, er
 		return resp, err
 	}
 
+	if verb != "GET" && verb != "HEAD" {
+		remaining := resp.Header.Get("Fastly-RateLimit-Remaining")
+		if remaining != "" {
+			val, err := strconv.Atoi(remaining)
+			if err == nil {
+				c.remaining = val
+			}
```

Comment:
The `val` variable isn't used after this block so might as well inline the conditional...

```suggestion
			if val, err := strconv.Atoi(remaining); err == nil {
				c.remaining = val
			}
```

Reply:
Done.